pip/tests/test_qsharp.py

```diff
@@ -257,6 +257,21 @@ def test_dump_operation() -> None:
             else:
                 assert res[i][j] == complex(0.0, 0.0)
 
+def test_run_with_noise_produces_noisy_results() -> None:
+    qsharp.init()
+    qsharp.set_quantum_seed(0)
```

Comment:
Do we need to set classical seed as well here to make it as deterministic as possible? (which seed does noisy simulation use?)

Reply:
We can add it for a peace of mind, but we don't need it. Pauli noise uses quantum seed, as it should (noise is quantum). Classical seed is used for things that don't involve quantum, such as DrawRandomInt, etc. And there're no calls to those functions in the test.

Reply:
I looked around and other tests set the seed they need to set (without trying to being "as deterministic as possible"). So, I think I'll keep my code.